src/libraries/System.Globalization.Extensions/tests/IdnMapping/IdnMappingIdnaConformanceTests.cs

```diff
@@ -21,6 +21,7 @@ public class IdnMappingIdnaConformanceTests
         /// There are some others that failed which have been commented out and marked in the dataset as "GETASCII DOES FAILS ON WINDOWS 8.1"
         /// Same applies to Windows 10 >= 10.0.15063 in the IdnaTest_9.txt file
         [Fact]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/58708", typeof(PlatformDetection), nameof(PlatformDetection.IsBrowser), nameof(PlatformDetection.IsMonoAOT), nameof(PlatformDetection.IsWindows))]
```

Comment:
how does this work? `PlatformDetection.IsBrowser` and `PlatformDetection.IsWindows` can't be both true at the same time.

Reply:
Added `PlatformDetection.IsBrowserOnWindows`: `public static bool IsBrowserOnWindows => IsBrowser && Path.DirectorySeparatorChar == '\\';`

Open to suggestions for a better way to do this.

Reply:
> Open to suggestions for a better way to do this.

I'm not sure if it's "better"... how about User-Agent sniffing ;-)

---

But more seriously: why does the host platform leak into how the app behaves?  Is something host-platform specific leaking through our tooling?